cmd/addimport_test.go

```diff
@@ -554,4 +558,5 @@ func TestAddImport_SameName(t *testing.T) {
 	require.Equal(t, "stream.database", string(ac.Imports[1].Subject))
 	require.True(t, ac.Imports[1].IsStream())
 	require.Equal(t, ts.GetAccountPublicKey(t, "B"), ac.Imports[1].Account)
+	require.True(t, ac.Imports[1].AllowTrace)
 }
```

Comment:
Would it make sense to add the tests similar to what you did for service export here for stream import. I am referring to TestAddServiceExportWTracing and TestAddStreamExportWTracing.

Reply:
No because those prompts won't show unless the import/export is the right kind.

Reply:
ok